src/renderer.js

```diff
@@ -271,7 +279,7 @@ class gltfRenderer
 
         if (this.parameters.useIBL)
         {
-            this.applyEnvironmentMap(gltf, envData, material.textures.length);
+            this.applyEnvironmentMap(gltf, envData, material.textures.length, material.defines.includes("MATERIAL_SHEEN 1"));
```

Comment:
`material.defines.includes("MATERIAL_SHEEN 1")` this is not explicit enough and can fail easily if the string changes